source/config/configSpec.py

```diff
@@ -249,6 +249,10 @@
 
 [development]
 	enableScratchpadDir = boolean(default=false)
+
+[featureFlag]
+	# 0:default, 1:yes, 2:no
+	cancelExpiredFocusSpeech = integer(0, 2, default=0)

```

Comment:
Why not make this an option like winConsoleUIA?

Reply:
So that I don't have to have to map strings to translated strings. The value matches the index of the translated string in a list. This simplifies the GUI creation.